.github/workflows/ci-notebooks.yml

```diff
@@ -24,15 +24,15 @@ jobs:
         with:
           path: taxi.csv
           # update cache only if notebooks require it to be changed
-          key: hashFiles("examples/tutorial/jupyter/execution/pandas_on_ray/local/**")
+          key: hashFiles("examples/tutorial/jupyter/**")
       # replace modin[all] with .[all] in the tutorial requirements file since we need
       # Modin built from sources
-      - run: sed -i 's/modin/./g' examples/tutorial/jupyter/execution/pandas_on_ray/requirements.txt
+      - run: sed -i 's/modin/./g' examples/tutorial/jupyter/requirements.txt
```

Comment:
We should have a single reqs file for each execution.

Reply:
I think you are right, there is no OmniSci package in pypi, so OmniSci notebooks need conda config. Separate reqs files are used.